tests/outputs/a_expense/expenses.yaml

```diff
@@ -0,0 +1,25 @@
+2003sb_google:
+	expense_type: business
```

Comment:
the indentation is almost certainly wrong here.  Make sure that this matches the actual output.

Also the travel expense is much larger so please could you build the test for that?  Please look in my new_presentation.py script for this (you can basically copy-paste it so it shouldn't take too long). Let me know if you need that.

Reply:
Thank you, i will add the test for it.

Reply:
I added an example of a travel expense present on the slack channel.